src/components/ItemCard.jsx

```diff
@@ -0,0 +1,29 @@
+import DefaultItemMd from "../assets/images/item-default-md.png";
+import DefaultItemSm from "../assets/images/item-default-sm.png";
+
+const ItemCard = ({ item }) => {
+  const { images, name, price, favoriteCount } = item;
```

Comment:
💊 제안
item 라는 객체로 props를 받기보다 images, name... 이렇게 받으시는게 더 좋아보여요.
지금 해당 컴포넌트에서 다른 props를 받는것이 아니라서 객체로 받으실 필요가 없어보입니다~

```suggestion
const ItemCard = ({ images, name, price, favoriteCount  }) => {
// 호출부: item.map((item) => <ItemCard key={name} {...item}/>)
```